oracle/src/watcher.js

```diff
@@ -155,8 +155,20 @@ async function isWorkerNeeded() {
   }
 }
 
+async function isShutdowned() {
+  logger.debug('Checking current shutdown state in the DB')
+  const isShutdown = (await redis.get(config.shutdownKey)) === 'true'
+  logger.debug({ isShutdown }, 'Read shutdown state from the DB')
+  return isShutdown
+}
+
 async function main({ sendToQueue, sendToWorker }) {
   try {
+    if (await isShutdowned()) {
+      logger.info('Oracle watcher was suspended via the remote shutdown process')
```

Comment:
Is my understanding correct and if the service was shutdown this message will be output in the log periodically? Does it make sense to output it only once to not flood the logs with the same message?